libs/utils/src/lib/button-with-menu/button-with-menu.component.ts

```diff
@@ -27,20 +27,26 @@ export class ButtonWithMenuComponent implements OnDestroy {
     private _vcr: ViewContainerRef,
```

Comment:
I think this prop needs a more explicit name

Reply:
Usually '_vcr' is used, but sometimes I met 'container' name. How do you think, is it gonna be better?)

Reply:
vcr sounds good enough, prob no `_` prefix. But I'd search across the app and make it consistent.